week-10/10-tech-lab.adoc

```diff
@@ -0,0 +1,228 @@
+= Lab 10: Security, and Governance, Risk, and Compliance
+
+== Objectives
+
+In this lab, you will gain some hands on experience in vulnerability scanning and remediation.
+
+Objectives:
+
+* Understand a "vulnerability" and how it can be detected
+* Understand how a vulnerability may be fixed or remediated
+
+== Overview
+
+Working individually, you will run an "inside-out" vulnerability scan and a simple "outside-in" penetration scan on a virtual machine.  
+
+The testing will be performed by downloading a vulnerability scanner and running it on your server.
+
+You will research the results and recommend at least one fix that can be implemented.
+
+You will fix at least two vulnerabilities and participate in group and class discussion about how this relates to the broader topics of governance, risk, and compliance.
+
+
+=== Getting started
+
+Start up a Virtual Machine, login to your personal server, perform an update and install lynis.
```

Comment:
please mention via Google cloud console

Reply:
OK.  Do you want me to show how to create a VM in the Google Cloud Console?  Do you want me to refer to Lab 01 if students can't remember how to create a VM from the command line?